proposals/templates/proposals/proposal_pdf.html

```diff
@@ -84,10 +84,9 @@
 
 {% block page_header %}
   <div id="page-header">
-    {% blocktrans with title=proposal.title reference_number=proposal.reference_number submitter=proposal.created_by.get_full_name trimmed %}
-      FETC-GW - <em>{{ title }}</em> (referentienummer {{ reference_number }}, ingediend door {{ submitter }})
+    {% blocktrans with title=proposal.title reference_number=proposal.reference_number submitter=proposal.created_by.get_full_name reviewing_committee=proposal.reviewing_committee.name trimmed %}
+      FETC-GW - <em>{{ title }}</em> (referentienummer {{ reference_number }}-{{reviewing_committee}}, ingediend door {{ submitter }})
```

Comment:
Hate to be that guy... But conventionally we place the committee _before_ the rest of the refnum.

Sorry, should've clarified that in my original comment